### ISSUE TITLE
Sandboxie Plus 1.0.10 ui crash when show warning

### ISSUE THREAD

**Issue by vitidev** [sandboxie-plus/Sandboxie#1621]
### What happened?

in version 1.0.9 sandboxed application does not start and shows warning

    15:48:55.985   GothicMod.exe (13284): SBIE2224 Sandboxed program has crashed: GothicMod.exe [Game_Gotic] 

But in version 1.0.10, when sanboxie try to show this message, it crashes sandboxie UI

Without isolation (green icon) the warning doesn't appear at all, so it doesn't crash.
But with standard isolation (yellow icon) and higher - a warning window button appears on the Windows taskbar, to which it is impossible to switch - and then it completely crashes UI

deleting the lines from the ini one by one, it was possible to find the problematic line 

    HideOtherBoxes=y

After removing it, I began to see warning without UI crashing

### What is your Windows edition and version?

Windows 10 Pro x64 21H1

### In which Windows account you have this problem?

A local or Microsoft account without special changes.

### Please mention any installed security software

Kaspersky Internet Security 21.3.10.39 (disabled)

### What version of Sandboxie are you running?

Sandboxie Plus 1.0.10 x64

### Is it a regression?

Sandboxie Plus 1.0.10 x64

### In which sandbox type you have this problem?

In a Standard isolation sandbox (yellow sandbox icon).

### Is the sandboxed program also installed outside the sandbox?

No, it is not installed in the real system.

### Can you reproduce this problem on an empty sandbox?

I can confirm it also on an empty sandbox.

### Sandboxie.ini configuration

```shell
[GlobalSettings]
Template=VirtualDesktopManager
Template=RpcPortBindings
Template=7zipShellEx
Template=RoboForm
Template=WindowsRasMan
Template=WindowsLive
Template=Kaspersky
Template=OfficeLicensing
FileRootPath=D:\Sandbox\%SANDBOX%
ActivationPrompt=n
SeparateUserFolders=y
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
EditAdminOnly=n
ForceDisableAdminOnly=n
ForgetPassword=n
NetworkEnableWFP=n
EnableObjectFiltering=n
EnableWin32kHooks=y

[test]
Enabled=y,viti
ConfigLevel=9
BorderColor=#00ffff,ttl,6
NotifyInternetAccessDenied=n
CopyLimitKb=9999152
OpenWinClass=*
OpenDefaultClsid=y
UnrestrictedSCM=n
ProtectRpcSs=n
ExposeBoxedSystem=n
HideOtherBoxes=y
BoxNameTitle=n
ClosePrintSpooler=y
ClosedFilePath=!<InternetAccess>,InternetAccessDevices
```

**Comment by DavidXanatos** [comment]
This should have been fixed with the latest build 1.0.11 please test and confirm

**Comment by vitidev** [comment]
v 1.0.11
For some reason it looks like 

    11:21:30.689   GothicMod.exe (6120): SBIE2101 OpenProcess (C0000022) access=00121410 initialized=1   
     .... 110 repeats
    11:21:30.720   GothicMod.exe (6120): SBIE2224 Sandboxed program has crashed: GothicMod.exe [Game_Gotic]   

whereas in 1.0.9 there was only the last line, but the gui does not crash

**Comment by DavidXanatos** [comment]
yes perfect that is expected,
the issue with HideOtherBoxes=y
is also known and will be fixed in 1.1.x as its a larger change and requires a test release before a final one.

**Comment by DavidXanatos** [comment]
https://github.com/sandboxie-plus/Sandboxie/issues/1603
